src/main/java/org/jenkinsci/plugins/GitLabSecurityRealm.java

```diff
@@ -221,24 +220,25 @@ public Object unmarshal(HierarchicalStreamReader reader, UnmarshallingContext co
         }
 
         private void setValue(GitLabSecurityRealm realm, String node, String value) {
-            switch (node.toLowerCase()) {
-                case "clientid":
-                    realm.setClientID(value);
-                    break;
-                case "clientsecret":
-                    realm.setClientSecret(value);
-                    break;
-                case "gitlabweburi":
+            switch (node) {
```

Comment:
Since all the cases that are being tested are English language strings, why not set the locale to English in the `node.toLowerCase()` call rather than risk that some caller was passing something that differed only by the case of the characters?